### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by Croitoru**
finished endpoints

### CODE REVIEW

File: ProjectR/TruEM-FE/src/app/_models/bank.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Account } from "./account";
+
+export interface Bank {
+    id: number;
+    name: string;
+    accounts: Account[];
+    Payable: boolean;
```
Comment (mihai-gritcan) [inline ProjectR/TruEM-FE/src/app/_models/bank.ts:7]: When serialize .NET Objects to Json, property names are lowercase

File: ProjectR/TruEM-FE/src/app/companies/add-employee/add-employee.component.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    secret: ''
+    };
+  }
+
+  ngOnInit(): void {
+  }
+
+  addCompanyEmployee(): void {
+    this.companyService.addEmployee(this.id, this.employee)
+      .subscribe();
```
Comment (mihai-gritcan) [inline ProjectR/TruEM-FE/src/app/companies/add-employee/add-employee.component.ts:92]: you can do some actions (eg: you'll be sure that the employee was or wasn't effectivelly persisted) inside the subscription when the callback success or error is acted ;)

**Review by mihai-gritcan — APPROVED** [review]

File: ProjectR/TruEM-FE/src/app/_models/bank.ts (hunk 1 above)
Comment (Croitoru) [reply ProjectR/TruEM-FE/src/app/_models/bank.ts:7]: Implemented

**Comment by Croitoru** [comment]
Implemented remarks as general. Applied changes where the same problems were observed

**Pull request merged**